cms/djangoapps/contentstore/views/exception.py

```diff
@@ -0,0 +1,18 @@
+"""
+A common module for managing exceptions. Helps to avoid circular references
```

Comment:
It appears to me that the only reference to CertificateValidationError is in views/certificates.py (and there are no usages of CertificateException beyond CertificateValidationError). Can we just put the exception class directly in that file?